### ISSUE TITLE
Hyperparamters in training

### ISSUE THREAD

**Issue by Bang2hen1iu** [Seed3D/Puppeteer#21]
Hi, would you please share some details about the hyperparameters used during training? Like the number of epochs, the learning rate, and weight decay factors of the optimizer. 
I am following the implementation details and part of the training components in MeshAnythingv2, trying to reproduce the training procedure. 
I train the model with the standard cross-entropy loss under a batch size of 512 on 4 48GB GPUs. But I observe a saturated convergence of loss after only 200 epochs of training. The loss degrades from 5.0 to approximately 1.2, but it is still too high and cannot generate any meaningful skeleton. 

If it is possible, I would also like to ask about the approximate magnitude of the loss value for the pretrained models.  Thanks a lot.

**Comment by ChaoyueSong** [comment]
Hi, we trained the skeleton model for 2,000 epochs on 8 A100 GPUs with a per-GPU batch size of 64. The learning rate was set to 1e-4 and decayed to 6e-5 using a cosine scheduler. The loss started above 4 and decreased to approximately 0.12 (I don't recall the exact number, but of course smaller than 1).

**Comment by Bang2hen1iu** [comment]
Did you use the pretrained weights from OPT-350M for finetuning or training from scratch? 
I've just finished a 2000 epochs of training a OPT-350M model from scratch to predict skeletons. 
The minimal value of loss reaches 0.7 and the generated skeletons is more meaningful, but the quantitative results are still far from the reported results (for testing on ArticulateXL-2.0, the J2J metric is 5.0+ vs. 3.0 reported in the paper). 
The only thing I missed in training is the sequence shuffling scheduling, which may not be able to cause such a large gap.

**Comment by Assld** [comment]
@ChaoyueSong Hello! Can you provide some more training parameter information for skinning? Thank you!
